### DISCUSSION TITLE
Welcome to MLM2PRO-GSPro-Connector Discussions!

### DISCUSSION THREAD

**Discussion by springbok** [springbok/MLM2PRO-GSPro-Connector#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rxbeaulieu-commits** [comment]
I have installed open api for mevo plus and gspro for the web cam putting. I have everything hooked up but when I go to gspro to practice and see if it works nothing happens when I'm hitting a ball. When I go back to see if everything is still green the gspro connect column has an yellowish DR box bellow connected. Can anyone help me I have deleted and installed so many times I'm ready to give up. I am new to this and I'm trying to learn as I go.  Any help would be appreciated. Thanks in advance
